lib/gitlab/request.rb

```diff
@@ -32,7 +32,8 @@ def self.parse(body)
 
     # Decodes a JSON response into Ruby object.
     def self.decode(response)
-      JSON.load response
+      return JSON.load response if response
+      {}
```

Comment:
Can you change this to `response ? JSON.load(response) : {}`?

Reply:
I just pushed this modification